src/main/java/uk/gov/digital/ho/hocs/search/application/queue/SearchConsumer.java

```diff
@@ -101,6 +105,15 @@ public void configure() {
                 .when(simple("${property.type} == '" + EventType.CASE_TOPIC_DELETED + "'"))
                 .to(DELETE_TOPIC_QUEUE)
                 .endChoice()
+                .when(simple("${property.type} == '" + EventType.SOMU_ITEM_CREATED + "'"))
+                .to(CREATE_SOMU_ITEM_QUEUE)
+                .endChoice()
+                .when(simple("${property.type} == '" + EventType.SOMU_ITEM_DELETED + "'"))
+                .to(DELETE_SOMU_ITEM_QUEUE)
+                .endChoice()
+                .when(simple("${property.type} == '" + EventType.SOMU_ITEM_UPDATED + "'"))
+                .to(UPDATE_SOMU_ITEM_QUEUE)
```

Comment:
Not for now, but we might wanna think about refactoring this to as it's getting quite long. Also, I reckon we'll be adding to this consumer a lot more in the future.

Reply:
Yeah I was thinking that, needs some work on best way to do it 👍🏼